core/src/main/java/tc/oc/pgm/command/ListCommand.java

```diff
@@ -59,6 +59,37 @@ public void list(CommandSender sender, Audience viewer, Match match) {
                 Integer.toString(getSize(match.getPlayers(), false)), TextColor.GREEN)));
   }
 
+  @Command(
```

Comment:
This command is most definitely for community.

Reply:
Will do 👍